### DISCUSSION TITLE
Typifying distribution

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#97] [people; relating]
Tom drafted decision tree to help make distribution choices. For the inventory flows, it might look like:
distribution | pros and cons
--|--
Normal | Pros: convenient, analytically tractable for the Kalman filter, ubiquitous.Cons: nonzero likelihood for impossible negative outcomes. Tails may be too light.
Lognormal | Pros: easy to specify noise proportional to the data.Cons: impossible to have 0 events, hard to mentally flip from arithmetic to geometric mean.
Poisson | Pros: simple derivation, easy to specify, includes 0Cons: discrete, no control over variance.
Neg Binomial (Poisson-Gamma mixture) | Pros: allows overdispersion, includes 0Cons: discrete, no possibility of underdispersion, limited control over 0 frequency

I also get the feeling that there's some generalized family that would encompass the above, with comprehensible parameterization in terms of reference std dev, variance-scale proportionality and 0-boundedness.

Angie prefers explicit heterogeneity expression + generic representation
- negative binom < gamma * poisson
- gamma * poisson < normal mixture * Poisson

McElreath (in Statistical rethinking) used lognormal for measurement, beta for 0-1 distribution, half-normal for non-negative distribution as below:

$$
\begin{aligned}
h_t & \sim \log -\operatorname{Normal}\left(\log \left(p_H H_t\right), \sigma_H\right) \\
\ell_t & \sim \log -\operatorname{Normal}\left(\log \left(p_L L_t\right), \sigma_L\right) \\
H_1 & \sim \text { Log-Normal }(\log 10,1) \\
L_1 & \sim \text { Log-Normal }(\log 10,1) \\
H_{T>1} & =H_1+\int_1^T H_t\left(b_H-m_H L_t\right) \mathrm{d} t \\
L_{T>1} & =L_1+\int_1^T L_t\left(b_L H_t-m_L\right) \mathrm{d} t \\
\sigma_H & \sim \operatorname{Exponential}(1) \\
\sigma_L & \sim \operatorname{Exponential}(1) \\
p_H & \sim \operatorname{Beta}\left(\alpha_H, \beta_H\right) \\
p_L & \sim \operatorname{Beta}\left(\alpha_L, \beta_L\right) \\
b_H & \sim \operatorname{Half-Normal}(1,0.5) \\
b_L & \sim \operatorname{Half-} \operatorname{Normal}(0.05,0.05) \\
m_H & \sim \operatorname{Half}-\operatorname{Normal}(0.05,0.05) \\
m_L & \sim \operatorname{Half-} \operatorname{Normal}(1,0.5)
\end{aligned}
$$

Distribution statements (with ~) are: (HL: hare and lynx)
lognormal: prob observed HL pelts, prior for initial HL population
exponential: prior for HL measurement dispersion
beta: prior for HL trap probability
half-normal: prior HL birth and death rate

The following from https://github.com/Data4DM/BayesSD/discussions/76#discussioncomment-4046702, in the end, it is not distribution, but the threshold (binning the region).

---
```
flexible tests using experimental priors
input: (structure, numeric) + managerial knowledge • choose test quantities: (log likelihood + log prior penalty) is default and modeler can add other test quantities • choose setting: different experimental priors for estimated, policy parameters
output: numeric and graphical diagnostics
```
Depending on the character of distribution (e.g.tail thickness), results are different.
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/30194633/202738424-0dcd79be-5b59-40c9-848c-4ba97859f358.png">

**Comment by tomfid** [comment]
What does the "distribution" here refer to - priors on structural parameters, or the measurement error model?

**Reply by hyunjimoon** [reply]
Realization of the "prior knowledge" concept. Hard and soft constraints could be one https://betanalpha.github.io/assets/case_studies/prior_modeling.html#31_Hard_Verses_Soft_Containment. Michael recommended later chapters of the book "[Good thinking](https://www.amazon.com/Good-Thinking-Foundations-Probability-Applications/dp/0486474380/ref=sr_1_6?keywords=good+thinking&qid=1668799882&sr=8-6)" for prior modeling.

**Comment by hyunjimoon** [comment]
From "In Pursuit of Perfect portfolio" which suggests four branches (risk aversion, income, spending,  environment) to build customized portfolio. What standards could we have for classify modelers (topology of modeler desire space?)

  ![image](https://user-images.githubusercontent.com/30194633/203860500-6022beaa-0043-44b7-9073-e034b59d6d43.jpeg)

**Reply by hyunjimoon** [reply]
+ Narrative
![image](https://user-images.githubusercontent.com/30194633/203860930-889c2a7e-d423-45dd-8931-1cf79509b948.jpeg)
